genesis/engine/bvh.py

```diff
@@ -177,6 +183,16 @@ def build(self):
         self.build_radix_tree()
         self.compute_bounds()
 
+    @ti.func
+    def filter(self, i_a, i_q):
+        """
+        Default filter function that always returns False.
```

Comment:
`Default filter function` default filter of what? In which context this function is being used and why it could be overwritten? This is not clear.